handbook/company/development-groups.md

```diff
@@ -185,12 +185,11 @@ Designs have usually gone through multiple rounds of revisions, but they could e
 
 
 #### Sub-tasks
-The simplest way to manage work is to use a single user story issue, then pass it around between contributors/asignees as seldom as possible.  But on a case-by-case basis, for particular user stories and teams, it can sometimes be worthwhile to invest additional overhead in creating separate **unestimated sub-task** issues ("sub-tasks").
+The simplest way to manage work is to use a single user story issue, then pass it around between contributors/asignees as seldom as possible.  But on a case-by-case basis, for particular user stories and teams, it can sometimes be worthwhile to invest additional overhead in creating separate **sub-task** issues ("sub-tasks").
 
 A user story is estimated to fit within 1 sprint and drives business value when released, independent of other stories.  Sub-tasks are not.
 
 Sub-tasks:
-- are NOT estimated
```

Comment:
If 99% of subtasks, which also happen to include 98% of the points of a given story, are closed in a sprint, then those subtasks count for 0 points, even if they were estimated as hundreds of points each.

^ thats the reasoning for making these "NOT estimated" here.  Ok with me to change this if we can explain that somehow

Reply:
Got it. Agree; sub-tasks count for nothing and are only counted at the user story level. In practice, the MDM group estimates sub-tasks but only counts the points at the rolled-up user story level. No change is necessary if that process matches jives with the docs.

Reply:
💯